package.json

```diff
@@ -100,12 +100,11 @@
       }
     }
   },
-  "dependencies": {
-    "@playcanvas/observer": "^1.3.6"
-  },
   "devDependencies": {
     "@playcanvas/eslint-config": "^1.0.16",
+    "@playcanvas/observer": "^1.3.6",
```

Comment:
Y'know, I _think_ this is redundant because the observer is a dependency of PCUI. So I suspect you can remove it completely.

Reply:
observer must remain a dependency (not dev-dep), so I've moved it back.